libsolidity/Compiler.h

```diff
@@ -41,6 +41,9 @@ class Compiler: private ASTConstVisitor
 	bytes getAssembledBytecode() { return m_context.getAssembledBytecode(m_optimize); }
 	bytes getRuntimeBytecode() { return m_runtimeContext.getAssembledBytecode(m_optimize);}
 	void streamAssembly(std::ostream& _stream) const { m_context.streamAssembly(_stream); }
+	/// Getters for compiler contexts. Only for testing purposes.
```

Comment:
yes we can. It's just exposing smaller part of the CompilerContext which may be better.